### PR TITLE
Integrate JSR 375 RI (soteria) in glassfish 5.

### LINKED ISSUE #21735: Integrate JSR 375 RI (soteria) in glassfish 5.
Integrate JSR 375 RI [soteria](https://github.com/javaee-security-spec/soteria) in GlassFish.

### PR DESCRIPTION
**Pull request by sameer-pandit**
Fixes #21735

### CODE REVIEW

File: main/appserver/pom.xml [hunk 1]
```diff
@@ -133,6 +134,7 @@
         <javax.management.j2ee-api.version>1.1.1</javax.management.j2ee-api.version>
         <guava.version>13.0.1</guava.version>
 	<bean.validator.cdi.version>2.5.0-b06</bean.validator.cdi.version>
+        <soteria.version>1.0-m05</soteria.version>
```
Comment (yaminikb) [inline main/appserver/pom.xml]: Please move this line to line 85 so that API and implementation versions are located together.

**Review by yaminikb — DISMISSED** [review]

**Comment by sameer-pandit** [comment]
@glassfishrobot Run CI tests please

**Comment by rdebusscher** [comment]
API and RI are updated to version 1.0-b06

groupId and artifactId are changed for the API project. See 
https://github.com/javaee-security-spec/security-api/commit/e3f1d34a02dfe7ba3505c69b4b4d85496f483600 for the updated name.

**Comment by sameer-pandit** [comment]
@glassfishrobot Run CI tests please

**Comment by sameer-pandit** [comment]
@glassfishrobot Run CI tests please.

**Review by yaminikb — APPROVED** [review]

**Comment by sameer-pandit** [comment]
@glassfishrobot Run CI tests please

**Review by vinayvishal — APPROVED** [review]
Looks good.

**Comment by sameer-pandit** [comment]
@glassfishrobot Run CI tests please

**Pull request merged**